waggle-dance-core/pom.xml

```diff
@@ -287,6 +287,7 @@
         <artifactId>spring-boot-maven-plugin</artifactId>
         <version>${spring-boot-maven-plugin.version}</version>
         <configuration>
+          <layout>ZIP</layout>
```

Comment:
Can you provide an explanation of what exactly this achieves? Perhaps you could unzip the .tgz that we produce with and without this change and run `tree` over the files/folders so we can see? (I'd try this out myself but I don't have the time right now!)